modules/statics/src/coins/adaTokens.ts

```diff
@@ -8,10 +8,10 @@ export const adaTokens = [
     'a7678172-84b3-4c7e-ac46-9875d23a1cb7',
     'tada:water',
     'Test ADA Token',
-    6,
+    0, // Tokens are not divisible - https://cardano-ledger.readthedocs.io/en/latest/explanations/token-bundles.html
```

Comment:
This will cause the issue, we use the smallest unit to calculate the total amount.